### DISCUSSION TITLE
Larger or adjustable text size for turn info during navigation

### DISCUSSION THREAD

**Discussion by quicklybeforeitstoo** [organicmaps/organicmaps#4172] [Ideas]
I understand that text placement and scaling across screen sizes is not always easy, and there are some other suggestions about UI sizing already. I think it's worth pointing out that that the name / exit number of the next turn is one of the most important things to read at a short glance, so having this be larger would be valuable even if nothing else is adjusted.

What specific scaling would be a personal preference, but for a phone screen I think having this text field take up 25% of the screen would be appropriate, as this is what I am always struggling to read quickly, more so than the map.

**Comment by biodranik** [comment]
That is a good idea (although 25% can be a bit too large in some cases). Please find and comment on the relevant issues. The ideal approach should draw some "good enough" size with normal system font sizes, and use larger fonts if a user selected larger fonts in the device's settings.
